geowebcache/core/src/main/java/org/geowebcache/mime/ApplicationMime.java

```diff
@@ -70,6 +72,9 @@ public class ApplicationMime extends MimeType {
     static Set<ApplicationMime> ALL =
             ImmutableSet.of(bil16, bil32, json, topojson, geojson, utfgrid, mapboxVector);
 
+    private static final List<String> binaryFormats =
```

Comment:
```suggestion
    private static final List<String> BINARY_FORMATS =
```

(yes I know this file is a mess and it does not consistently use uppercase for constants but... that's what our [coding convention](https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names) says we should be doing).

Leave the others as they are, changing the casing would mean breaking client code, unfortunately (we could do, but it would require going through deprecation too...)